### PR TITLE
Rename to NumFetch + Minor Changes

### PR DESCRIPTION
**Pull request by The-ZmaZe**
- Rename to NumFetch according to NumX scripts (i.e. NumCraft)

- Added RAM and Screen

- Add numfetch() function to allow uses in shell

- Correct name for NumWorks N0110

### CODE REVIEW

File: numfetch.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+__author__ = ("Tim-ats-d@github.com", "The-ZmaZe@github.com")
```
Comment (Tim-ats-d) [inline numfetch.py:1]: Not necessary, right?

Comment (Tim-ats-d) [inline numfetch.py:1]: Not necessary, right?

File: numfetch.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  "Uptime: " + str(time.monotonic()),
+  "Shell: " + "µpython " + uname["version"],
+  "Resolution: 320x240",
+  "MCU: " + ("STM32F730V8T6" if uname["machine"] == "NumWorks N0110" else "STM32F412"),
+  "Memory: " + ("8Mo" if uname["machine"] == "NumWorks N0110" else "1Mo"),
+  "RAM: " + ("256Ko" if uname["machine"] == "NumWorks N0110" else "1Mo"),
+  "Screen: " + ("IPS-LCD" if uname["machine"] == "NumWorks N0110" else "TFT-LCD")
+)
+
+def numfetch(ascii:str="omega"):
```
Comment (Tim-ats-d) [inline numfetch.py]:
```suggestion
def numfetch(ascii: str = "omega"):
```
PEP8 fix.

**Review by Tim-ats-d — CHANGES_REQUESTED** [review]
Thank for this pull request.

File: numfetch.py (hunk 1 above)
Comment (The-ZmaZe) [reply numfetch.py:1]: Sure

**Pull request merged**